gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -979,68 +1066,111 @@ def main(args):
     data_type = args.data_type
     contig = args.contig
     joint_included = args.joint_included
-    resources = get_export_resources(overwrite, data_type, test)
+    for_joint = data_type == "joint"
+
+    resources = get_export_resources(
+        overwrite,
+        data_type,
+        test,
+    )
 
     if contig and test:
         raise ValueError(
             "Test argument cannot be used with contig argument as test filters"
             " to chr20, X, and Y."
         )
-
+    if data_type == "joint" and joint_included:
+        raise ValueError(
+            "Joint data type and joint included cannot both be true. joint_included"
+            "is only used for exomes and genomes data types."
+        )
     try:
         if args.validate_release_ht:
             logger.info("Running release HT validation...")
```

Comment:
```suggestion
            logger.info("Running %s release HT validation...", data_type)
```
I think this will be helpful looking at the results.